test_tools/src/test_tools/learn_objects.py

```diff
@@ -0,0 +1,73 @@
+#! /usr/bin/env python
+
```

Comment:
@PetervDooren more of a general question: why can/do you not use (a slightly altered) inspect action?

Reply:
This is a slightly altered inspect action. More specifically it is a stripped down version of segment_objects.
Initially I only wanted to perform segmentation on the objects, since classification would take a trained network to train a trained network. I ended up using more of the segment objects function than I would like.
The main reason to not just use the sement_objects smach state is that we don't want to move the head every time.